### PR TITLE
fix(clients-driving-license): updating driving license endpoints

### PR DESCRIPTION
**Pull request by arnorhs**
The service has been updated to use the same casing for dev/prod, so this should address that change.

I'm not going to add automerge until RLS has allowed access to our client ID, and then I'll add it

Edit: It's open now, and I am able to call some of the endpoints now, so I believe it should be fixed

## Checklist:

- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] Formatting passes locally with my changes
- [x] I have rebased against main before asking for a review

### CODE REVIEW

File: libs/api/domains/driving-license/src/lib/drivingLicense.service.ts [hunk 1]
```diff
@@ -166,6 +166,7 @@ export class DrivingLicenseService {
         kennitala: nationalId,
       },
     )) as unknown) as string
+    console.log({ statusStr })
```
Comment (pshomov) [inline libs/api/domains/driving-license/src/lib/drivingLicense.service.ts:169]: Remove? or use logging lib

**Review by pshomov — CHANGES_REQUESTED** [review]
LGTM, small cleanup needed

**Review by albinagu — APPROVED** [review]
lgtm

File: libs/api/domains/driving-license/src/lib/drivingLicense.service.ts (hunk 1 above)
Comment (arnorhs) [reply libs/api/domains/driving-license/src/lib/drivingLicense.service.ts:169]: oops, ty

Comment (arnorhs) [reply libs/api/domains/driving-license/src/lib/drivingLicense.service.ts]: should be fixed

**Review by jeremybarbet — APPROVED** [review]
Nice! LGTM

**Review by pshomov — APPROVED** [review]
LGTM

**Pull request merged**